mne/realtime/tests/test_fieldtrip_client.py

```diff
@@ -8,70 +8,155 @@
 import subprocess
 import warnings
 import os.path as op
+import contextlib
+import socket
+
+import numpy as np
+from numpy.testing import assert_array_equal, assert_allclose
+import pytest
+
+from mne import Epochs, find_events, pick_types
+from mne.io import read_raw_fif
+from mne.utils import requires_neuromag2ft
+from mne.utils import run_tests_if_main
+from mne.realtime import FieldTripClient, RtEpochs
+from mne.externals.six.moves import queue
 
-from nose.tools import assert_true, assert_equal
+from .test_mockclient import _call_base_epochs_public_api
 
-import mne
-from mne.utils import requires_neuromag2ft, run_tests_if_main
-from mne.realtime import FieldTripClient
-from mne.externals.six.moves import queue
+# Set our plotters to test mode
+import matplotlib
+matplotlib.use('Agg')  # for testing don't use X server
```

Comment:
shouldn't this be imported before everything else? Because mne also imports matplotlib